components/homepage/howItWorks.tsx

```diff
@@ -0,0 +1,137 @@
+import Image from "next/image";
+// How it works section of the homepage
+export default function HowItWorks() {
+    return (
+      <div id="HowItWorks" className="w-full 2xl:py-20 py-16 px-24">
+        <div className="2xl:text-6xl text-5xl font-semibold text-center 2xl:mb-32 mb-28">
+          How It Works
+        </div>
+        {/* Steps Section */}
+        <div className="w-full 2xl:w-[90rem] mx-auto flex flex-col 2xl:space-y-32 space-y-24">
+          {/* Create organization section */}
+          <section className="flex items-center justify-between">
+            {/* Text */}
+            <div className="w-[30rem]">
+              <div className="2xl:text-4xl text-3xl font-semibold">
+                Create your <p className="text-maroon inline">organization</p>
+              </div>
+              <div className="text-gray-500 mt-4 text-lg">
+                Create your organization and invite your team members to join.
+                If you are your own organization, create an organization to
```

Comment:
Weird to read create an organization twice essentially

Reply:
I'll do this tmr but what wording do you suggest